ocaml/xapi/cert_distrib.ml

```diff
@@ -565,10 +565,32 @@ let collect_ca_certs ~__context ~names =
 
 (* This function is called on the pool that is incorporating a new host *)
 let exchange_ca_certificates_with_joiner ~__context ~import ~export =
+  let open Certificates in
```

Comment:
Another option to avoid long module names would be to `let module C = Certificates in ... C.pem_of_string ..` such that names are still qualified.